Angular/src/app/add-capability/add-capability.component.spec.ts

```diff
@@ -1,14 +1,21 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { AddCapabilityComponent } from './add-capability.component';
+import { ReactiveFormsModule } from '@angular/forms';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { DataService } from '../_services/data.service';
 
 describe('AddCapabilityComponent', () => {
   let component: AddCapabilityComponent;
   let fixture: ComponentFixture<AddCapabilityComponent>;
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [ AddCapabilityComponent ]
+      declarations: [ AddCapabilityComponent ],
+      imports: [ ReactiveFormsModule ],
+      providers: [ NgbActiveModal,
+                   DataService 
```

Comment:
formatting is preferred like this: 
https://github.com/sapphireduffy/KainosCareerLattice/pull/39/files#diff-5ecc1ba348aae4edd8273433f83d029aR17